pyproject.toml

```diff
@@ -85,6 +85,10 @@ addopts = [
   "--allow-hosts=localhost,::1,stripe",
   "--durations=20",
   "--numprocesses=auto",
+  # Disable ddtrace for tests
+  "-p no:ddtrace",
+  "-p no:ddtrace.pytest_bdd",
+  "-p no:ddtrace.pytest_benchmark",
```

Comment:
> Given the previous results - I think only disabling trace plugins in the tests is enough.

Then should this be the only change necessary, and restore the other changes proposed?